src/gui/layout/qgslayoutviewtoolselect.cpp

```diff
@@ -130,6 +130,12 @@ void QgsLayoutViewToolSelect::layoutPressEvent( QgsLayoutViewMouseEvent *event )
     return;
   }
 
+  if ( selectedItem->isLocked() )
```

Comment:
I think this logic actually belongs in QgsLayout::layoutItemAt, when ignoreLocked is true we should be ignoring items which are part of a locked group and instead returning the NEXT item below at the specified position.

Right now this just aborts the selection, which isn't correct -- it should transparently select the next item below.

Reply:
Thanks for the suggestion. I applied it and it works as expected even with stacked locked group over an unlocked item.